understanding/22/target-size-minimum.html

```diff
@@ -40,15 +40,15 @@ <h2>In brief</h2>
       <dl>
           <dt>Objective</dt><dd>Make controls easier to activate</dd>
           <dt>Author task</dt><dd>Ensure targets meet a minimum size or have sufficient spacing around them</dd>
-          <dt>Key beneficiaries</dt><dd>Users with some physical disabilities</dd>
+          <dt>Key beneficiaries</dt><dd>Users with limitations to fine motor input</dd>
       </dl>
     </section>
     
     <section id="intent">
         <h2>Intent of Target Size (Minimum)</h2>
 
-        <p>The intent of this Success Criterion is to help ensure targets can be easily activated without accidentally activating an adjacent target. When targets are small, it is difficult for users with hand tremors and those who have difficulty with fine motor movement to activate them accurately when there are other targets that are too close. Providing sufficient size, or sufficient spacing between targets, will reduce the likelihood of accidentally activating the wrong control.</p>
-
+        <p>The intent of this Success Criterion is to help ensure targets can be easily activated without accidentally activating an adjacent target.  Users with dexterity limitations and those who have difficulty with fine motor movement find it difficult to accurately activate small targets when there are other targets that are too close.  Providing sufficient size, or sufficient spacing between targets, will reduce the likelihood of accidentally activating the wrong control.</p>
+        <p>Disabilities addressed by this requirement include hand tremors, spasticity, and quadriplegia.  Some people with disabilities use specialized input devices instead of a computer mouse or trackpad.  Typically these types of input device do not provide as much accuracy as mainstream pointing devices.  Meeting this requirement also ensures that touchscreen interfaces are easier to use.</p>
```

Comment:
i'd remove the "two spaces after a full stop" typewriter convention here. it's not rendered in web content, and unnecessary in the age of non-monospaced fonts

Reply:
@patrickhlauke the double space after full stop was for my own benefit -- since I am editing in monospace.  Yes, HTML ignores those regardless.

But, as an aside, I am of the opinion that "unnecessary in the age of non-monospaced fonts" is not entirely correct.  Can you point me to a web browser that provides a little extra space between sentences (and after colons), as compared to periods ending abbreviations?  That bit of extra whitespace is very common in print publications and its typography tooling chain, and it really helps with readability.

Reply:
> Can you point me to a web browser that provides a little extra space between sentences (and after colons), as compared to periods ending abbreviations?

not sure i understand the question. all browsers, per spec, collapse multiple spaces into a single space (unless it's inside a `<pre>` preformatted element) when displaying/rendering HTML content.

Reply:
I'm going to chime in here.
 @patrickhlauke, agreed that the extras space after the period when published will disappear, as will all the potentially weird line spaces, indents, etc., that some of us use to more easily work with the info in our code viewers. 
But that doesn't mean that _while working in a monospace code viewer_, some of us don't derive benefit from that 'unpreserved' formatting.
I think you two are at risk of sending this comment thread into the weeds :) but I think it might be an idea to work out between you 'offline' when it's okay to not trim spaces.

Reply:
not precious about the double spaces. leave them, remove them. just for consistency, nowhere else in this same file do we have double spaces, so purely from a code consistency point of view...